Test.C

```diff
@@ -581,3 +590,86 @@ void encryptionStore(struct Player *playerPtr, struct Encryption encryptionStock
         return;
     }
 }
+
+/******************************************************************************
+* Function:    addScore
+* Description: Example of curl calls to add player score
+* Parameters:  playerName, score, seed, version
+* Return:      void
+******************************************************************************/
+void addScore(char playerName, int score, int seed, int version){
+  CURL *curl;
+  CURLcode res;
+  
+  /* Creates the fields in which the scores are saved to the website */
+  sprintf(char *fields, "player_name=%s&score=%d&seed=%d&version=%d", &playerName, &score, &seed, &version);
```

Comment:
fields is a string or character array in C. `char *fields` is not how you initiate an array. Please initiate the array outside of the `sprintf` function. This is why you are getting `fields` was not declared in this scope on line 674